lib/util/RestConnection.js

```diff
@@ -193,9 +193,9 @@ module.exports = class RestConnection {
   /**
    * @private Merge headers and automatically append sessionHeaders if available
    *
-   * @param {*} [customHeaders={}]
-   * @param {*} mergeHeaders
-   * @returns
+   * @param {Object} [customHeaders={}]
+   * @param {Boolean} mergeHeaders
+   * @returns {Object} containing desired headers merged into single object
```

Comment:
Since you're here, could you change the JSDoc types to primitives too? Think all of these can be lower-cased.

Reply:
What needs to be done here?

Reply:
As discussed, change `{Boolean}` to `{boolean}`

Reply:
Done.